Cython/Compiler/Nodes.py

```diff
@@ -39,6 +39,13 @@
     _py_int_types = (int, long)
 
 
+IMPLICIT_CLASSMETHODS = set()
+if sys.version_info[0:2] >= (3, 6):
+    IMPLICIT_CLASSMETHODS.add("__init_subclass__")
+if sys.version_info[0:2] >= (3, 7):
+    IMPLICIT_CLASSMETHODS.add("__class_getitem__")
```

Comment:
Open to suggestions on a better place for these definitions to exist and/or better way to express this.

Reply:
This is definitely the wrong way to check the version of Python - the version used to run Cython isn't necessarily the same as the version of Python the code will run in.

I still don't think this needs version checks - there's no harm in supporting this even on Python 2.7. Nobody should be using these attributes for anything else since I'm fairly sure Python reserves all unused special method names for its own use.